cloudstate-kotlin-support/src/main/kotlin/io/cloudstate/kotlinsupport/initializers/ConfigIntializer.kt

```diff
@@ -0,0 +1,8 @@
+package io.cloudstate.kotlinsupport.initializers
+
+class ConfigIntializer {
+    var host: String = "0.0.0.0"
+    var port: Int = 8088
+    var loglevel: String = "DEBUG"
```

Comment:
Should it be DEBUG by default?

Reply:
Fixed https://github.com/cloudstateio/kotlin-support/pull/8/commits/16d1cc844f931b925b9d7a16048ee63864f5f22b